docs/faq.md

```diff
@@ -126,3 +138,11 @@ Make sure that your firewall configuration allows the connection and try using t
 
 ## How can I transfer the results of skin simulation to the final mesh?
 Make use of the [AdnSkinMerge](maya/skin_merge) deformer to blend animated and simulated meshes into a single final mesh. Launch the Skin Merge tool, select the final mesh, add the animated and simulated target meshes and click on Create. Then you only need to paint the blend weight to define the influence of the simulated mesh targets over the animated ones. You can find more information [here](maya/simple_setup#adnskinmerge).
+
+## How can I change the logger level?
+
+By default, AdonisFX solvers print log messages to the console if their level is `Info`, `Warning`, or `Error`. However, `Debug` log messages can be enabled to assist with troubleshooting. To enable `Debug` messages, set the environment variable `ADN_LOGGER_LEVEL` to `0`. To disable `Debug` messages again, set the environment variable to `1`. Note that for these changes to take effect, the AdonisFX plugin must be reloaded.
+
+## How can I control the multithreading of AdonisFX solvers?
+
+AdonisFX nodes use multithreading based on a threshold that determines the minimum number of elements required for parallel processing. If the number of elements to process exceeds this threshold, multithreading will be applied, otherwise, the code will execute sequentially. This threshold is set to `16` by default, but can be adjusted by modifying the value of the `ADN_MIN_PARALLEL_SIZE` environment variable. Modify this value carefully, as it can improve performance in some cases but may also reduce it depending on your CPU. Note that for these changes to take effect, the AdonisFX plugin must be reloaded.
```

Comment:
I think this is not true... This variable is read at each initialization. Could you please double
`Note that for these changes to take effect, the AdonisFX plugin must be reloaded.`

Reply:
Carlos is correct. ADN_LOGGER_LEVEL gets read on load. But in this case it's on init

Reply:
Correct, I've double checked it and, ADN_MIN_PARALLEL_SIZE can be modified in the script editor (with python) and it will be read on init.